### DISCUSSION TITLE
CompreFace 0.6.0

### DISCUSSION THREAD

**Discussion by pospielov** [exadel-inc/CompreFace#589] [General]
## Features

* Redesign. Nicer look and support of small resolutions
* Plugins on UI. Now you can test how age, gender, and landmarks plugins work from UI.
* Mask recognition plugin. We added a new plugin that returns if the person wears a mask and if yes - if it's worn correctly
* Subject endpoints. This functionality adds more options to manage your face collections
* Download endpoints. Now you can download saved images

## Fixes

- API node synchronization 

## Migration from 0.5.1 version

* If you start CompreFace 0.6.0 after using the 0.5.1 version, it will run migration scripts. Migration can take quite a long time, so please be patient. 
* "Delete All Examples of the Subject by Name" endpoint now returns the number of deleted examples instead of the list of them.
* The age and gender plugin response format was changed and now contains a probability field.

## How to install:
1. Download CompreFace_0.6.0.zip archive or run:
`wget -q -O tmp.zip 'https://github.com/exadel-inc/CompreFace/releases/download/v0.6.0/CompreFace_0.6.0.zip' && unzip tmp.zip && rm tmp.zip`
2. To start CompreFace run:
`docker-compose up -d`
3. Open in your browser: http://localhost:8000/login
4. To stop properly run: 
`docker-compose stop`
5. To start again run: 
`docker-compose start`

More detail install guide(and Windows instructions) is on our [readme](https://github.com/exadel-inc/CompreFace#getting-started-with-compreface)﻿ page.

<hr /><em>This discussion was created from the release <a href='https://github.com/exadel-inc/CompreFace/releases/tag/v0.6.0'>CompreFace 0.6.0</a>.</em>

**Comment by jakowenko** [comment]
The UI updates are great. Very refreshing look. Keep up the great work!

**Reply by pospielov** [reply]
Thanks! Even more UI improvements are coming next release :)

**Reply by HarisMohre** [reply]
Hello, 

For some reason , the Download API`s are responding with 404. Can you let me know what could be the problem?

The below is the API:

-X GET "http://localhost:8000/api/v1/recognition/faces/<image_Id>/img" -H "x-api-key: <api_key>"

**Reply by pospielov** [reply]
Hi, have you changed the <image_Id> and <api_key> to your valid values?

**Reply by mczarsty** [reply]
hi @jakowenko! i am one of the designers for CompreFace. i am currently working on a ux survey  - i'd love to chat quickly about your thoughts on where we could potentially improve CompreFace, if you are interested? if you are i can set up a skype or whatsapp chat :)
